api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -35,7 +35,7 @@ package object models {
   )
 
   implicit lazy val ArtifactFormat: Format[Artifact] = (
-    (__ \ "url").format[String] ~
+    (__ \ "uri").format[String] ~
```

Comment:
Yet the [schema](https://github.com/dcos/metronome/blob/master/api/src/main/resources/public/api/v1/schema/jobspec.schema.json#L48) specifies the property to be `uri`, and imo the released schema is the source of truth in this case. I'd be totally ok with renaming it to `url` to be more specific about the intent, but this would be a breaking change.